moodle_rb.gemspec

```diff
@@ -11,12 +11,12 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/jobready/moodle-rb'
   spec.license       = 'MIT'
 
-  spec.files         = `git ls-files -z`.split("\x0")
+  spec.files         = Dir["lib/**/*"] + %w(README.md LICENSE Rakefile)
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
+  spec.test_files    = Dir['spec/**/*']
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'bundler', '~> 1.3'
+  spec.add_development_dependency 'bundler', '~> 2'
```

Comment:
The Ruby 2.7 image installs a 2.x version of bundler. While it can be overridden by manually installing the older 1.x version and then using it for the bundle command, I don't believe there is reason not to just up the bundler version here.